cl/users/management/commands/cl_account_management.py

```diff
@@ -57,29 +57,31 @@ def handle(self, *args, **options):
             print("* NO EMAILS SENT OR ACCOUNTS DELETED *")
             print("**************************************")
 
-    def delete_old_accounts(self):
-        """Find accounts older than roughly two months that have not been
-        confirmed, and delete them. Should be run once a month, or so.
+    def delete_old_accounts(self) -> None:
+        """Delete old, unused accounts
+
+        Delete accounts older than 60 days that lack a confirmed email
+        address and never logged in.
+
+        :return None
         """
         two_months_ago = now() - datetime.timedelta(60)
         unconfirmed_ups = UserProfile.objects.filter(
             email_confirmed=False,
             user__date_joined__lte=two_months_ago,
+            user__last_login=None,
             stub_account=False,
         )
 
         for up in unconfirmed_ups:
+            if self.options["simulate"]:
+                return
+
+            delete_user_assets(up.user)
```

Comment:
Is it necessary to call this method here? since we'll delete only accounts that never logged in. Is it not supposed that these kinds of accounts wouldn't have any of that kind of data?

Reply:
Good call. Removed.